src/Sylius/Bundle/PaymentBundle/Resources/config/routing/integrations/sylius.yaml

```
@@ -0,0 +1,9 @@
+sylius_webhook_payment_method:
+    path: /payment-method/{code}
+    defaults:
+        _controller: sylius.action.webhook_payment_request
+
+sylius_shop_webhook_payment_request:
+    path: /payment-request/{hash}
```

Should it be like so?
```suggestion
    path: /payment-requests/{hash}
```